### PR TITLE
Hw order

### PR DESCRIPTION
**Pull request by GradientAspen**

### CODE REVIEW

File: src/main/java/com/project/springbootproject/dto/order/OrderDtoStatus.java [hunk 1]
```diff
@@ -0,0 +1,9 @@
+package com.project.springbootproject.dto.order;
+
+import com.project.springbootproject.model.Order;
+import lombok.Data;
+
+@Data
+public class OrderDtoStatus {
+    private Order.Status status;
+}
```
Comment (MentorMA) [inline src/main/java/com/project/springbootproject/dto/order/OrderDtoStatus.java:9]: Why you create many small dto classes?

File: src/main/java/com/project/springbootproject/model/ShoppingCart.java [hunk 2]
```diff
@@ -29,13 +30,14 @@ public class ShoppingCart {
     private Long id;
 
     @OneToOne
-    @MapsId
+    //@MapsId was problem with ShoppingCartService getByUserId method
```
Comment (MentorMA) [inline src/main/java/com/project/springbootproject/model/ShoppingCart.java]: Commented code

File: src/main/java/com/project/springbootproject/repository/cartitem/CartItemRepository.java [hunk 3]
```diff
@@ -1,7 +1,14 @@
...
 
 import com.project.springbootproject.model.CartItem;
+import jakarta.transaction.Transactional;
 import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Modifying;
+import org.springframework.data.jpa.repository.Query;
 
 public interface CartItemRepository extends JpaRepository<CartItem,Long> {
+    @Transactional
+    @Modifying
```
Comment (MentorMA) [inline src/main/java/com/project/springbootproject/repository/cartitem/CartItemRepository.java]: why do you use this annotations?

**Review by MentorMA — CHANGES_REQUESTED** [review]

File: src/main/java/com/project/springbootproject/dto/order/OrderDtoStatus.java (hunk 1 above)
Comment (GradientAspen) [reply src/main/java/com/project/springbootproject/dto/order/OrderDtoStatus.java:9]: At the moment I find it more convenient to work with the returned type in this way

**Review by kozhukhovsky — APPROVED** [review]

**Pull request merged**